hazelcast/include/hazelcast/client/internal/socket/SocketFactory.h

```diff
@@ -56,8 +56,8 @@ namespace hazelcast {
                     spi::ClientContext &client_context_;
                     boost::asio::io_context &io_;
                     boost::asio::ip::tcp::resolver &io_resolver_;
+                    std::shared_ptr<boost::asio::ssl::context> ssl_context_;
```

Comment:
Does this compile without OpenSSL or `boost/asio/ssl/context.hpp`?

Reply:
fixed.